readthedocs/core/static-src/core/js/doc-embed/utils.js

```diff
@@ -15,4 +15,13 @@ const createDomNode = (nodeName, attributes) => {
     return node;
 };
 
-module.exports = { createDomNode };
+const domReady = (fn) => {
```

Comment:
I didn't notice this for the other function here, but a named function should be fine for both of these utility functions:

```suggestion
function domReady(fn) {
```